lib/autoInject.js

```diff
@@ -4,10 +4,10 @@ import arrayMap from 'lodash/_arrayMap';
 import clone from 'lodash/_copyArray';
 import isArray from 'lodash/isArray';
 
-var argsRegex =  /^function\s*[^\(]*\(\s*([^\)]*)\)/m;
+var argsRegex = /^(function[^\(]*)?\(?\s*([^\)=]*)/m;
 
 function parseParams(func) {
-    return  func.toString().match(argsRegex)[1].split(/\s*\,\s*/);
+    return func.toString().match(argsRegex)[2].trim().split(/\s*\,\s*/);
```

Comment:
The trim is necessary because it doesn't support spaces around the function name parameters, e.g. `function( x, y, z )`, and then there was a space after the final parameter.  This wasn't obvious until I added support for arrow functions like `x => y`.  I tried inventing a regex which coped with both functions and arrows, but failed, and the trim seemed like an easy workaround.

Alternatively, it could import a dedicated library, e.g.: https://www.npmjs.com/package/get-parameter-names